test/internal/constants/constants.go

```diff
@@ -0,0 +1,15 @@
+package constants
+
+const (
+	PlatformWindows    = "windows"
+	PlatformLinux      = "linux"
+	SnapshotterWindows = "windows"
+	SnapshotterLinux   = "windows-lcow"
+)
+
+func SnapshotterFromPlatform(platform string) string {
```

Comment:
Better to return the default snapshotter configured in containerd.toml like we do here: https://github.com/kevpar/cri/blob/windows_port/pkg/server/service_windows.go#L69, that way if we add new cimfs snapshotter this wouldn't have to change. (The platform still remains "windows" for cimfs)

Reply:
I updated the logic to handle unknow platforms, but I left it as is for now
I want to add the ability to ask containerd for all the snapshotters it has, but ill save that for the future, since I think it may be unnecessary and containerd will use the default on on its own...